frontend/public/components/secrets/create-secret.tsx

```diff
@@ -393,10 +414,14 @@ type ConfigEntryFormProps = {
   id: number;
   entry: Object;
   onChange: Function;
+  t: TFunction;
 };
 
```

Comment:
Instead of doing this, you can also import the type WithTranslation from react-i18next and do something like 
```
type ConfigEntryFormState = WithTranslation & {...};
```

You have to disable the linter when you import it: 
```
/* eslint-disable-next-line */
import { withTranslation, WithTranslation } from 'react-i18next';
```

Reply:
Removed the `t: TFunction` prop here, missed that. It looks now like this https://github.com/openshift/console/blob/ce0918bdd9c249aa6056ba90b7b40894d6618000/frontend/public/components/secrets/create-secret.tsx#L413-L422

I could not use `WithT` from `react-i18n` because I need an optional here to make the t not required for the component consumers.

Reply:
I tested it again and it works fine now. So I replaced `{ t?: TFunction }` with predefined type `WithT`:

https://github.com/openshift/console/blob/8c8ca211efcdb1c4bd9f71a15211ad998ef1b309/frontend/public/components/secrets/create-secret.tsx#L413-L422

Reply:
That's fine by me! Thanks so much. This is something that's come up when Sam's looked at my code before. Appreciate the extra work on your end.